keylime/cmd/convert_runtime_policy.py

```diff
@@ -123,9 +123,6 @@ def update_runtime_policy(
     policy: RuntimePolicyType, excludelist_path: Optional[str] = None, verification_keys: Optional[List[str]] = None
 ) -> RuntimePolicyType:
     if policy["meta"]["version"] < ima.RUNTIME_POLICY_CURRENT_VERSION:
-        print(
-            f"Provided policy has version {policy['meta']['version']}; latest policy has version {ima.RUNTIME_POLICY_CURRENT_VERSION}. Updating to latest version."
-        )
```

Comment:
That is a fair notice, why not to keep it?

Reply:
Because we may not upgrade it to v2 after trimming.